setup.py

```diff
@@ -22,7 +22,7 @@ def version_file(mode='r'):
     version=version(),
     author='apsliteteam, oznu',
     author_email='aps@odin.com',
-    packages=['osaapi'],
+    packages=['osaapi', 'apsapi'],
     url='https://aps.odin.com',
     license='Apache License',
     description='A python client for the Odin Service Automation (OSA) and billing APIs.',
```

Comment:
A python binding for Odin Automation and APS controller https://doc.apsstandard.org/7.1/api/rest/